setup.py

```diff
@@ -18,6 +18,7 @@
         "openreview-py",
         "ortools>=8.1.8487",
         "pytest",
+        "sortedcontainers>=2.4.0",
```

Comment:
what is this new library for?

Reply:
The new algorithm.

Reply:
The algorithm needs to maintain a sorted list of papers in order of priority, because at each step there will usually be multiple papers with top priority and we need to iterate over them. I used the SortedList class from this library to implement that.